### ISSUE TITLE
TypeScript implicit 'any'

### ISSUE THREAD

**Issue by alpha0010** [callstack/react-native-testing-library#708]
## Describe the bug

Updated to 7.2.0, and got:
```
node_modules/@testing-library/react-native/typings/index.d.ts:302:3 - error TS7020: Call signature, which lacks return-type annotation, implicitly has an 'any' return type.

302   (message?: string);
      ~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

## Expected behavior

Type checks run clean.

## Steps to Reproduce

1. Install latest version of testing library
2. `npx tsc`

## Versions

```
  npmPackages:
    @testing-library/react-native: ^7.2.0 => 7.2.0 
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.4 => 0.63.4 
    react-test-renderer: 16.13.1 => 16.13.1
```

**Comment by varwasabi** [comment]
Will open a PR for this shortly

**Comment by jk-eviivo** [comment]
Looks like it is already in master via this PR: https://github.com/callstack/react-native-testing-library/pull/697

**Comment by varwasabi** [comment]
Looks like it is already fixed in master via this PR: https://github.com/callstack/react-native-testing-library/pull/697

**Comment by thymikee** [comment]
published in 8.0.0-rc.1
